src/test/java/org/apache/commons/codec/net/BCodecTest.java

```diff
@@ -157,6 +158,7 @@ public void testDecodeObjects() throws Exception {
     }
 
     @Test
+    @Ignore("CODEC-280: Disabled strict decoding by default. The BCodec uses the default so this test does not fail the impossible cases.")
```

Comment:
@aherbert 
I think this test should be rewritten to match the new behavior, not `@Ignore`'d.

Reply:
The new behaviour is lenient. So that would be a behaviour change for BCodec and make this test redundant.

I think that the new strict decoding property can be added here too so users have an option.

Reply:
I'm all for making Base32 and Base64 carry the same option.

Reply:
Hi @garydgregory 

I received an email from github with another comment from you but cannot find it through the web interface so I'll repeat for clarity the comment was "I'm all for making Base32 and Base64 carry the same option."

This PR already has the public setStrictDecoding(boolean) for Base32 and Base64. The unresolved is whether the property should be added to BCodec as well. At the moment this test is ignored because BCodec cannot be made strict. So either:

1. Drop the test and BCodec is always lenient
2. Make BCodec always strict (this test will pass)
3. Make BCodec have configurable strict/lenient decoding

My option would be number 3.